Engine/src/Engine/SceneBuilder/Scene.h

```diff
@@ -24,6 +28,12 @@ namespace Engine
     public:
 
         #pragma region Entity Management
+
+        unsigned int m_fbo;
+        unsigned int textureColorbuffer;
```

Comment:
I know some of these are my commits, but those were under the impression that this branch would be discarded.
For Rendering specific things, please keep them out of the review since they're out of the scope of the PR

Reply:
I may be misunderstanding, but the m_fbo is being used in this pr. The pr includes functionality for the GUI and the render panel that uses the fbo.

Reply:
fbo stuff is now removed from this pr. The game window is still built to take in an fbo but uses a default value of 0 for the time being.